src/System.Net.Security/src/System/Net/SecureProtocols/_SecuritySafeHandles.Windows.cs

```diff
@@ -218,117 +218,6 @@ public new IntPtr DangerousGetHandle()
             throw NotImplemented.ByDesign;
         }
 #endif
-
-        #if false //TODO if not used in Nego Stream as well, please remove it.
```

Comment:
NegotiateStream can pull in the relevant code from tfs as in required. These are currently not used anywhere, so it does not make sense to keep it.